collections/_posts/2023-03-14-Deep-Label-Fusion.md

```diff
@@ -0,0 +1,172 @@
+
+ * The maps obtained are used to assign different weights to each atlas depending on the pixel considered
+
+## Label Fusion computation
+
+* Once the weight maps are acquired, the candidate segmentations $$S_i$$ are fused into initial consensus segmentation $$S^{init}$$ in 3 steps :
+  - Candidate segmentations are converted to one-hot encoding segmentations $$S^i=\left\{p^i_l,l=1,2,...,N_{label} \right\} $$
+  - Vote maps $$V^i=\left\{V^i_l,l=1,2,...,N_{label} \right\} $$ are computed by elementwise multiplying $$W^i$$ and $$S^i$$ for each labels and spatial location n
+  - For each label, the vote maps of all the atlases are average to generate the initial segmentation
+
+## Fine-tuning subnet
+
+* This subnet is a 4 level U-Net designed to correct remaining residual errors after label fusion 
+
+* Input : $$S^{init}$$ and associated coordinate maps
+* Output : Set of feature maps of the same size as $$S^{init}$$
+
+* A label-specific mask is then generated taking the union of all candidate segmentations and multiplying it by the corresponding channel of the output to get the final segmentation.
+
+> Note : This label-masking operation assumes that the truth label is contained inside the region that has atlas votes of that label
+
+
+# Data augmentation strategies
+
+* In addition to usual data augmentation (Random flips, rotations, patching, elastic deformation and additive gaussian noise), DLF has 3 dedicated data augmentation methods :
+
+  - A random selection of $$N_{Atlas}$$ is made for each target image. $$N_{Atlas}$$ can be superior to the total number of available datasets. This random selection allows for duplicate atlases to help handle similar votes, as well as making the network more robust to atlas variability
+
+  - An extended Random Histogram Shift (RHS) is applied to atlas & target independantly, which corresponds to a change in contrast between the atlas image and the target image. This helps the weighted voting subnet to be more sensitive to image structure rather than intensity distribution
+  
+  - In case of multi-modality (T1 & T2 here), **Modality Augmentation (ModAug)** is implemented. The idea is to randomly replace a modality during training with white noise to force the model to base its prediction on individual modalities
+
+
+
+# Experiments & Results
+
+## Datasets
+![](/collections/images/DeepLabelFusion/datasets.jpg) 
+Figure 3 : Different datasets used for the experiments
+
+## Cross-validation experiments
+
+* Comparison with other MAS algorithms with various label fusion methods (Majority Voting (MV), Spatially Varying Weighted Voting (SVWV), Joint Label Fusion (JLF))
+
+* Comparison with a 3D U-Net with a similar architecture as the fine-tuning model of DLF, as well as with nnUNet
+
+* Oracle(10%) results consider that a pixel is rightly segmented after registration if at least 10% of atlases have the correct segmentation. Those results are used to evaluate the upper bound of label fusion performance
```

Comment:
I would more clearly separate this Oracle with the baseline to which they compare themselves to.